_docs/deprecated_and_archived_modules.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: post
+title: Deprecated and Archived Modules
+date: 2019-11-29
```

Comment:
Will people see this post if it's so far in the past?

Reply:
it's not in the blog section: https://voxpupuli.org/docs/